app/components/Resource/Resource.js

```diff
@@ -51,7 +51,7 @@ class Resource extends Component {
                 </div>
               </div>
               <div className="org-desc">
-                <p>This is an organisation's long description{this.props.description}</p>
+                <p>{this.state.resource.long_description || this.state.resource.short_description || "Description of the resource"}</p>
```

Comment:
Pinged Derek on Slack, but if he doesn't respond by tomorrow, let's just change it to "No description available." Other than that, this looks good to merge.